.jenkins/pytorch/build.sh

```diff
@@ -167,3 +173,48 @@ if [[ "$BUILD_TEST_LIBTORCH" == "1" ]]; then
   make VERBOSE=1
   popd
 fi
+
+# Test XLA build
+if [[ "${JOB_BASE_NAME}" == *xla* ]]; then
+  # TODO: Move this to Dockerfile.
```

Comment:
Yes please ;)

Reply:
I will remove these in a following PR I guess, once this test runs stably in our CI, I will try to update our docker image. :P